_functions.sh

```diff
@@ -2028,6 +2046,9 @@ do_start() {
   if [ ! -z "${DEPLOYMENT_LDAP_LINK}" ]; then
     echo_info "LDAP URL  : ${DEPLOYMENT_LDAP_LINK}"
   fi
+  if [ ! -z "${DEPLOYMENT_IFRAMELY_LINK}" ]; then
+    echo_info "IFRAMELY URL  : ${DEPLOYMENT_IFRAMELY_LINK}"
+  fi
```

Comment:
Do we need really this ? IMO it has be protected (eg hosting 403 error due to cors policies)

Reply:
will not really foccus on that, no critical data on the container